dashboard/app/views/congrats/index.html.haml

```diff
@@ -16,4 +16,5 @@
 #congrats-container
 
 - content_for(:head) do
+  = stylesheet_link_tag 'css/petition', media: 'all'
```

Comment:
I am worried about these styles affecting other things on the page. I don't see any obvious problems right now, but some of the styles in that file use fairly general selectors: https://github.com/code-dot-org/code-dot-org/blob/cf1a749cc4942ee40ad0e5a424b87fbce6c43ee0/apps/style/code-studio/petition.scss#L73-L83

What I would suggest is wrapping any style which does not already have "petition" in the name inside petition.scss inside `.petition-block { ... }`. for example:
```
.petition-block {
  input.grey_input {
    width: 150px;
  }

  span.grey_input {
    width: 160px;
  }
  ...
}
```
if it's easiest, you could even wrap most/all of the file contents in one such block.

otherwise, additions to the congrats page, or header or footer menus, could unexpectedly match these styles and cause the page to look different than expected.

Reply:
Ah yes, good call here––will update!

Reply:
hi Meg, sorry to go back on this, but per [slack](https://codedotorg.slack.com/archives/C0T0PNR0D/p1654792960315299?thread_ts=1654789948.903569&cid=C0T0PNR0D) convo with Maddie, please try including this directly into the component instead of including it "globally" here. if this works, then you should no longer need to wrap all the styles in the outer block.

Reply:
Yep, good call! Didn't push those commits before heading out. I kept the wrapping styles for added safety, but I don't have to if you think readability is better now that we aren't using the styles outside the petition